.github/ISSUE_TEMPLATE/propose-a-new-sample.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Propose a new sample
+about: Suggest an idea for a new sample
+title: ''
+labels: new sample request
+assignees: ''
+
+---
+
+<!-- NOTE: Given the maintenance cost of the large number of samples that are already in this repository, we are unlikely to add new samples. However, if you have an idea for a sample that is worth the additional maintenance burden, please propose it below -->
```

Comment:
maybe we just take this template away, and suggest people propose new samples in the [discussions](https://github.com/firebase/functions-samples/discussions)?

Reply:
Yeah I think that's a good idea.

Reply:
👍 removed this file